Gemfile

```diff
@@ -58,11 +57,11 @@ group :plugins do
   gem 'haml'
   gem 'kramdown'
   gem 'less', '~> 2.6', platforms: :ruby
-  gem 'libv8', platforms: :ruby
```

Comment:
@ddfreyne it seems it breaks CI build

```
Errno::ENOENT: No such file or directory @ apply2files - /home/runner/work/nanoc/nanoc/vendor/bundle/ruby/2.6.0/gems/libv8-node-16.10.0.0-x86_64-linux/vendor/v8/x86_64-linux-musl/libv8/obj/libv8_monolith.a
```

Reply:
Yeah — apparently on Ruby 2.6 only though, which is EOL so I plan on dropping Ruby 2.6 support rather than fixing this.